draft-ietf-quic-manageability.md

```diff
@@ -945,68 +945,72 @@ deliver packets between server and NAT. Reducing the timeout on UDP NATs might
+exhausted, it should drop the initial packets of a connection rather than
+risking breakage later. As QUIC is based on UDP and some devices are known to block
+UDP, QUIC deployments are recommended to implement a fallback to TCP
+{{?I-D.ietf-quic-applicability}}. Therefore it is often preferable for QUIC
+to fail and fallback during connection establishment rather than persist with a
+connection that might be unstable or experiencing
+sudden connection errors and time outs later during the connection.
 
 ### "Helping" with routing infrastructure issues
 
 Concealing client address changes in order to simplify operational routing
 issues will mask important signals that drive security mechanisms, and
 therefore opens QUIC up to various attacks.
 
-One challenge in QUIC deployments that want to benefit from QUIC's migration
-capability is server infrastructures with routers and switches that direct
-traffic based on address-port 4-tuple rather than connection ID. The use of
-source IP address means that a NAT rebinding or address migration will deliver
-packets to the wrong server. As all QUIC payloads are encrypted, routers and
-switches will not have access to negotiated but not-yet-in-use CIDs. This is a
-particular problem for low-state load balancers. {{QUIC_LB}} addresses this
-problem proposing a QUIC extension to allow some server-load balancer
-coordination for routable CIDs.
-
-It seems that a NAT anywhere in the front of such an infrastructure setup could
-save the effort of converting all these devices by decoding routable connection
-IDs and rewriting the packet IP addresses to allow consistent routing by legacy
-devices.
-
-Unfortunately, the change of IP address or port is an important signal to QUIC
-endpoints. It requires a review of path-dependent variables like congestion
-control parameters. It can also signify various attacks that mislead one
-endpoint about the best peer address for the connection (see section 9 of
-{{QUIC-TRANSPORT}}). The QUIC PATH_CHALLENGE and PATH_RESPONSE frames are
-intended to detect and mitigate these attacks and verify connectivity to the
-new address. This mechanism cannot work if the NAT is bleaching peer address
-changes.
+While QUIC's migration capability makes it possible for an server to survive
+address changes, this does not work if the routers or switches in the server
+infrastructures reply on address-port 4-tuple as a NAT rebinding or address
+migration will cause packets to be delivered to the wrong server. {{QUIC_LB}}
+addresses this problem proposing a QUIC extension to allow server-load balancer
+coordination to routable CIDs.
+
+An alternative, potentially simpler approach would seem to be the use of NAT in front
+of such an infrastructure setup. However, hiding information about the change
+of the IP address or port conceals important, and security relevant information
+from QUIC endpoints, and as such would facilitate amplification attacks (see
+section 9 of {{QUIC-TRANSPORT}}).
```

Comment:
Rather than phrase this as a viable alternative (as the lead-in text implies), make it clear that this is non-viable.

> It might seem possible to use a NAT that uses connection IDs to provide a stable address 4-tuple.  This might work if servers cooperate to inform the NAT of all connection IDs that might be used for each connection.  However, hiding address changes from a server prevents the server from validating new addresses, which would facilitate amplification attacks (see {{Section 9 of QUIC-TRANSPORT}}).